### PR TITLE
Adds Django REST Framework 3.5.x compatibility

### LINKED ISSUE #143: Add DRF 3.5 support

### PR DESCRIPTION
**Pull request by kciredor**
Fixes #143

### CODE REVIEW

File: dynamic_rest/filters.py [hunk 1]
```diff
@@ -665,4 +669,8 @@ def get_valid_fields(self, queryset, view):
                 if not getattr(field, 'write_only', False) and
                 not field.source == '*' and field_name in valid_fields
             ]
+
+        if 'get_template_context' == caller:
```
Comment (aleontiev) [inline dynamic_rest/filters.py]: This seems hacky / brittle -- why should the caller change the return value? Is there any other way we can retain compatibility that is caller-agnostic?

Comment (kciredor) [inline dynamic_rest/filters.py]: I agree, if your PR #138 would finally get merged :-) then I could drop this hack entirely!

Comment (aleontiev) [inline dynamic_rest/filters.py]: #140 is merged

**Comment by kciredor** [comment]
Nice one @aleontiev - I have updated my PR accordingly. Ready to go :+1:

**Comment by aleontiev** [comment]
Awesome, looks good

**Pull request merged**